src/components/spreadsheet/spreadsheet.xml

```diff
@@ -23,18 +23,10 @@
             <HeaderGroupContainer layers="rowLayers" dimension="'ROW'"/>
           </div>
           <div class="o-group-grid overflow-hidden">
-            <Grid
-              sidePanelIsOpen="sidePanel.isOpen"
-              exposeFocus="(focus) => this._focusGrid = focus"
-            />
+            <Grid exposeFocus="(focus) => this._focusGrid = focus"/>
           </div>
         </div>
-        <SidePanel
-          t-if="sidePanel.isOpen"
-          onCloseSidePanel="() => this.closeSidePanel()"
-          component="sidePanel.component"
-          panelProps="sidePanel.panelProps"
-        />
+        <SidePanel t-if="sidePanel.isOpen"/>
```

Comment:
Maybe better to have the t-if in the component directly ?

Reply:
you'd end up with an empty div that might be subject to css styling, no?

Reply:
Probably possible to put the t-if on the top-most div, and nothing is displayed then ? Not sure.

Reply:
Yes indeed. But that would mean the `SidePanel` component would still be created. Not critical, but since we already need the store, this t-if doesn't harm